Dockerfile

```diff
@@ -2,24 +2,23 @@ FROM gliderlabs/alpine:3.6
 MAINTAINER Hypothes.is Project and contributors
 
 # Install system build and runtime dependencies.
-RUN apk-install ca-certificates python py-pip libpq \
-  collectd collectd-disk supervisor
+RUN apk-install ca-certificates python3 libpq collectd collectd-disk supervisor
```

Comment:
We install `supervisor` with `apk-install` like bouncer does (instead of with pip as h does) because we have `pip3` and `supervisor` doesn't support Python 3.